core/prelude-packages.el

```diff
@@ -37,6 +37,15 @@
 
 (add-to-list 'package-archives
              '("melpa" . "https://melpa.org/packages/") t)
+
+;; load the pinned packages
+(let ((prelude-pinned-packages-file (expand-file-name "prelude-pinned-packages.el" prelude-dir)))
+  (if (file-exists-p prelude-pinned-packages-file)
+      (load prelude-pinned-packages-file)
+    (message "Missing pinned packages file %s" prelude-pinned-packages-file)
```

Comment:
I wouldn't warn if the file is missing. Users might interpret this as an indication something's wrong, when it actually isn't.